src/js/common/pages/CampaignSupport/CampaignSupportPayToPromote.jsx

```diff
@@ -288,7 +291,31 @@ class CampaignSupportPayToPromote extends Component {
                     </CampaignSupportDesktopButtonPanel>
                   </CampaignSupportDesktopButtonWrapper>
                   <CampaignSupportMobileButtonWrapper className="u-show-mobile">
-                    <CampaignSupportMobileButtonPanel>
+                    {(isCordova()) ? (
+                      <CampaignSupportMobileButtonPanel>
+                        <Suspense fallback={<span>&nbsp;</span>}>
+                          <OpenExternalWebSite
+                            linkIdAttribute="submitPayToPromoteMobile"
+                            url={`${AppObservableStore.getWeVoteRootURL()}${this.getCampaignXBasePath()}pay-to-promote-process`}
+                            target="_blank"
+                            title="Yes, I'll chip in $3"
+                            body={(
+                              <Button
+                                classes={{ root: classes.buttonDefault }}
+                                color="primary"
+                                variant="contained"
+                              >
+                                Yes, I&apos;ll chip in $3&nbsp;
+                                <Launch
+                                  style={{
+                                  }}
+                                />
+                              </Button>
+                            )}
+                          />
+                        </Suspense>
+                      </CampaignSupportMobileButtonPanel>
+                    ) : (
                       <Button
```

Comment:
Hi @jpena925 I forgot when we met that you also have to wrap the Button in the "else" block with `<CampaignSupportMobileButtonPanel>`. You can also delete the "style" attribute in `<Launch`. Finally, OpenExternalWebsite should be included using Lazy, instead of a direct import.

Reply:
Hi @DaleMcGrew I made the changes here. Let me know if this works or if I should make a new PR. Thanks for the feedback and help!